examples/point_cloud/main.cpp

```diff
@@ -171,8 +171,9 @@ int main(int argc, char *argv[]) {
     }
 #endif
 
-    snprintf(p_xyz_frame_output_file_path, sizeof(p_xyz_frame_output_file_path),
-             "%s/%s", output_path, p_xyz_frame_output_file_name);
+    snprintf(p_xyz_frame_output_file_path,
```

Comment:
I think it would be much simpler to use std::string.

Reply:
Using std::string now.

Reply:
Using std::string now